src/mutable_data.rs

```diff
@@ -961,6 +962,16 @@ impl SeqEntryActions {
         &self.actions
     }
 
+    /// Converts the SeqEntryActions into a BTreeMap containing keys with their corresponding action
+    pub fn into_actions(self) -> BTreeMap<Vec<u8>, SeqEntryAction> {
+        self.actions
+    }
+
+    /// Gets the actions mutably
+    pub fn get_mut(&mut self) -> &mut BTreeMap<Vec<u8>, SeqEntryAction> {
+        self.actions.borrow_mut()
```

Comment:
Maybe I'm wrong but can't this simply be 

```rust
&mut self.actions
```

Reply:
One more comment: since already have a method called `actions`, this should be called `actions_mut` for consistency.